assets/control-plane/service-monitor-kubelet.yaml

```diff
@@ -131,6 +131,27 @@ spec:
       certFile: /etc/prometheus/secrets/metrics-client-certs/tls.crt
       insecureSkipVerify: false
       keyFile: /etc/prometheus/secrets/metrics-client-certs/tls.key
+  - bearerTokenFile: ""
+    honorLabels: true
+    interval: 5s
+    path: /metrics/slis
```

Comment:
We should have the discussion if we should collect these new metrics by default.

cc @dgrisonnet @jan--f